### DISCUSSION TITLE
Motion Detection - Person only on RLC-810A

### DISCUSSION THREAD

**Discussion by widewings007** [fwestenberg/reolink_dev#249] [Q&A] [answered]
Hi, is there a way I can set the motion to detection to person motion only as I am trying to control a light with this. At the moment there is a lot of motion detection and my light is turning on and off even though not needed.

Thanks for the help.

(Looks like this is solved already with the latest update - Thanks for making the modification)

**Comment by rysm83** [comment]
I too am looking to use the AI person detection on my camera.  I have RLC-520A that I have added via this integration.  Motion is working however the person motion detection is not.  It is always clear and not detection anything.  I would like to use this AI motion for automations like this one.  What I have done in the mean time is integrated DOODS object detection into HA.  So far it is working ok about 80-90% of the time.  What I have done is use a motion sensor to detect the motion then using DOODS do image processing of the camera to scan for a person.  If true then the light in the room turns on.  Once motion is not detected it once again scans the image for a person to keep the light on since someone is still in the room.  only when the image processing does not detect the person then the light turns off.   I think that using the AI with the reolink camera would make this simpler and not need to DOODS integration if anyone can help to get that working?

**Reply by cmpoyner** [reply]
If you manually set to vehicle/motion detection under push notifications and then use the integration to toggle notifications on/off your vehicle/person settings are changed to Any Motion.

Disregard, this appears to function properly after updating camera firmware to 3.0.0.494

**Reply by Teevolution** [reply]
I have the same problem. HA detect motion from all my camera but no the AI person, pet, car? I did have it working via person detection but it now does not. Has there been an update from Reolink that has stopped this working?

**Reply by de-salvatierra** [reply]
> If you manually set to vehicle/motion detection under push notifications and then use the integration to toggle notifications on/off your vehicle/person settings are changed to Any Motion.
> 
> Disregard, this appears to function properly after updating camera firmware to 3.0.0.494

My camera has firmware version 3.0.0.177_21012103. Where did you get version 3.0.0.494?

**Reply by cpainchaud** [reply]
https://reolink.com/download-center/

**Comment by de-salvatierra** [comment, marked as answer]
> If you manually set to vehicle/motion detection under push notifications and then use the integration to toggle notifications on/off your vehicle/person settings are changed to Any Motion.
> 
> Disregard, this appears to function properly after updating camera firmware to 3.0.0.494
My camera has firmware version 3.0.0.177_21012103. Where did you get version 3.0.0.494?

**Reply by cpainchaud** [reply]
https://reolink.com/download-center/
